suite/tests/CMakeLists.txt

```diff
@@ -2948,6 +2948,7 @@ else (UNIX)
     append_link_flags(security-win32.except-execution "/dynamicbase:no")
 
     tobuild(security-win32.except-guard security-win32/except-guard.c)
+    tobuild(security-win32.except-int2d security-win32/except-int2d.c)
```

Comment:
If the -guard test works, sure (cleanest as separate CL but ok either way) -- if it does I guess we just didn't test all of these on x64?  I was assuming they all had problems on x64.